components/Storyblok/SbBasicPage.tsx

```diff
@@ -20,31 +22,37 @@ export const SbBasicPage = ({
   blok: {
     title,
     hero,
+    heroImage: { filename, focus } = {},
+    subheading,
+    heroContent,
     content,
     ankle,
   },
   blok,
-}: SbBasicPageProps) => (
-  <div {...storyblokEditable(blok)}>
-    <Masthead />
-    <main id="main-content">
-      <div>
-        <CreateBloks blokSection={hero} />
-        <Hero heading={title} />
-        <CreateBloks blokSection={content} />
-        {getNumBloks(ankle) > 0 && (
-          <Image
-            width={2000}
-            height={40}
-            alt=""
-            loading="lazy"
-            src={getProcessedImage('https://a-us.storyblok.com/f/1005200/2000x40/c4777a4925/steve-johnson-cropped-2000x40-01.jpg')}
-            className="w-full"
-          />
-        )}
-        <CreateBloks blokSection={ankle} />
-      </div>
-    </main>
-  </div>
-);
+}: SbBasicPageProps) => {
+  const HeroContent = !!getNumBloks(heroContent) ? <CreateBloks blokSection={heroContent} /> : undefined;
+
+  return (
+    <div {...storyblokEditable(blok)}>
+      <Masthead />
+      <main id="main-content">
+        <div>
+          {!!getNumBloks(hero) ? (
+            <CreateBloks blokSection={hero} />
+          ) : (
+            <BasicHeroMvp
+              title={title}
+              subheading={subheading}
+              imageSrc={filename}
+              imageFocus={focus}
+              heroContent={HeroContent}
+            />
+          )}
```

Comment:
Render the BasicHeroMVP if no hero block is added - this is to prepare for the next phase where we'll have different heroes for initiative, theme and maybe other type of pages where they can be inserted as Storyblok components